test/unittest/tests/test_reconciliation_tracking.cpp

```diff
@@ -41,25 +23,3 @@ TEST_CASE("Reconciliation tracking: ovfl_discard_verbose", "[reconciliation]")
         REQUIRE(__ut_ovfl_discard_verbose(session, nullptr, nullptr, nullptr) == EINVAL);
     }
 }
-
-TEST_CASE("Reconciliation tracking: ovfl_discard_wrapup", "[reconciliation]")
-{
-    ConnectionWrapper conn(DB_HOME);
-    WT_SESSION_IMPL *session = conn.createSession();
-
-    WT_PAGE p;
-    memset(&p, 0, sizeof(p));
-
-    WT_PAGE_MODIFY m;
-    memset(&m, 0, sizeof(m));
-    p.modify = &m;
-
-    REQUIRE(__wt_ovfl_track_init(session, &p) == 0);
-
-    SECTION("handle empty overflow entry list")
-    {
-        REQUIRE(__ut_ovfl_discard_wrapup(session, &p) == 0);
```

Comment:
If we do end up removing this test after all, please check if this "\_\_ut_" function is still necessary. But hopefully we can still keep it 😀

Reply:
Done